ovos_bus_client/client/client.py

```diff
@@ -116,11 +116,13 @@ def on_error(self, *args):
             LOG.warning('Could not send message because connection has closed')
         elif isinstance(error, ConnectionRefusedError):
             LOG.warning('Connection Refused. Is Messagebus Service running?')
+        elif isinstance(error, ConnectionResetError):
+            LOG.warning('Connection Reset. Did the Messagebus Service stop?')
         else:
             LOG.exception('=== %s ===', repr(error))
+            self.emitter.emit('error', error)
```

Comment:
non-blocking, consider a try:except here to catch other possible exceptions. If we're only seeing ConnectionResetError associated with `self.emitter.emit()` in logs then it's probably not necessary.